lib/Drush/Command/InitCommand.php

```diff
@@ -0,0 +1,174 @@
+<?php
+namespace Drush\Command;
+
+/**
+ * @file
+ *   Set up local Drush configuration.
+ */
+
+use Drush\Log\LogLevel;
+
+class InitCommand extends \Robo\Tasks
```

Comment:
I see the `@option` annotations below on the individual methods, but should there be an annotation on the class too for command discovery? Or are those intended to be discovered by other means?